### ISSUE TITLE
Package no longer on CPAN/MetaCPAN?

### ISSUE THREAD

**Issue by ArcticSnowman** [sdondley/moo-google#22]
Package no longer on CPAN/MetaCPAN?

Have this been replaced by something else?

**Comment by sdondley** [comment]
Yes. Another guy who had a lot more interest in it than me took it on. I don't remember who it was or what it's called now. I think there may be some discussion here in the issue queue with him.
